pallets/xyk/src/lib.rs

```diff
@@ -863,29 +919,23 @@ impl<T: Config> Pallet<T> {
 			),
 		);
 
-		let mut rewards_claimed: i128 = 0;
-		if LiquidityMiningUserClaimed::<T>::contains_key((user.clone(), &liquidity_asset_id)) {
-			rewards_claimed =
-				Self::liquidity_mining_user_claimed((user.clone(), &liquidity_asset_id));
+		let mut rewards_to_be_claimed: u128 = 0;
+		if LiquidityMiningUserToBeClaimed::<T>::contains_key((user.clone(), &liquidity_asset_id)) {
```

Comment:
use ::get instead contains key, it will either return value or 0 by default

Reply:
replaced by get